bedrock/mozorg/views.py

```diff
@@ -254,15 +254,12 @@ def home_tweets(locale):
 def home(request, template='mozorg/home/home.html'):
     locale = l10n_utils.get_locale(request)
 
-    # en-US gets new template (or variation)
+    # en-US gets new template
+    # not using locale-specific templates naming scheme yet as page/test results
```

Comment:
Incidentally, I'm not sure if it's currently possible to use a locale-specific template for en-US only, since it defaults to en-US as a fallback? At least, last time I tried to do this it didn't seem to work, but perhaps I was doing something wrong.